src/main/java/edu/cornell/kfs/sys/util/LoadFileUtils.java

```diff
@@ -44,4 +45,20 @@ public static byte[] safelyLoadFileBytes(File file) {
         }
         return safelyLoadFileBytes(file.getAbsolutePath());
     }
+    
+    /**
+     * Returns the string contents of a file.  If the file can not be loaded, an empty string is returned.
+     * @param fullyQualifiedFileName
+     * @return
+     */
+    public static String safelyLoadFileString(String fullyQualifiedFileName) {
+        try {
+            byte[] fileByteArray = safelyLoadFileBytes(fullyQualifiedFileName);
+            String formattedString = new String(fileByteArray);
```

Comment:
I know the original method only used the single-arg String constructor, but you may want to consider updating this to use a constructor that also accepts a charset.

Reply:
ok